src/main/java/io/github/jhipster/jpa2jdl/NewRunner.java

```diff
@@ -0,0 +1,124 @@
+package io.github.jhipster.jpa2jdl;
+
+import org.kohsuke.args4j.CmdLineException;
+import org.kohsuke.args4j.CmdLineParser;
+import org.kohsuke.args4j.Option;
+
+import javax.persistence.Entity;
+import java.io.BufferedWriter;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.lang.annotation.Annotation;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.Arrays;
+import java.util.Enumeration;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+import java.util.jar.JarEntry;
+import java.util.jar.JarFile;
+
+public class NewRunner {
+
+    private ReverseJPA2JDL jpa2jdl = new ReverseJPA2JDL();
+
+    @Option(name="--package", required = true)
+    private String rootPackage;
+
+    @Option(name="--archive", required = true)
+    private String archiveFile;
+
+    @Option(name="--archiveRoot", required = false)
```

Comment:
This should be considered as a tool, rather than software to generate jdl from itself, hence I think there is no point to use it like it was before. What default value should be used for pointing jar archive? I cannot imagine default value here.